scripts/setup-mel-builddir

```diff
@@ -124,7 +126,7 @@ configure_for_machine () {
     echo 'BBLAYERS = "\' >> $BUILDDIR/conf/bblayers.conf
     layersfile=$(mktemp setup-mel-builddir.XXXXXX)
 
-    layerpaths="*:*/*:meta*/*/*:$MELDIR/*:$MELDIR/*/*:$MELDIR/meta*/*/*"
+    layerpaths="*:*/*:meta*/*/*:$MELDIR/*:$MELDIR/*/*:$MELDIR/meta*/*/*${DIRS:+:}$DIRS"
```

Comment:
`DIRS` is 1) too vague a name, 2) inconsistent with the case of our existing commandline argument variables, and 3) isn't set to a default value along with our other commandline option variables, meaning its value would also be picked up from the user's environment if set, which is particularly risky given the generic variable name. Please name it something more appropriate and set a default value, thanks!